components/automate-ui/src/app/pages/api-token/details/api-token-details.component.ts

```diff
@@ -62,6 +81,31 @@ export class ApiTokenDetailsComponent implements OnInit, OnDestroy {
       .subscribe((id: string) => {
         this.store.dispatch(new GetToken({ id }));
       });
+
+    combineLatest([
+      this.store.select(iamMajorVersion),
+      this.store.select(iamMinorVersion)]).pipe(takeUntil(this.isDestroyed))
+        .subscribe(([major, minor]: [IAMMajorVersion, IAMMinorVersion]) => {
+          this.isMajorV1 = major === 'v1';
+          this.isMinorV1 = minor === 'v1';
+        });
+
+    combineLatest([
+      this.store.select(allProjects),
+      this.store.select(getAllProjectStatus)
+    ]).pipe(
+      takeUntil(this.isDestroyed),
+      filter(([_, pStatus]: [Project[], EntityStatus]) => pStatus !== EntityStatus.loading),
+      filter(() => !!this.token),
+      map(([allowedProjects, _]) => {
+        allowedProjects
+          .forEach(p => {
+            this.projects[p.id] = {
```

Comment:
💭 Would projects ever be removed from this? Looks like we're only adding/overwriting keys here -- I might be missing something.

Reply:
Good catch; fixed.